File: async-scala-client/src/main/scala/io/swagger/client/model/SalesCalculatedTaxSummaryForService.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class SalesCalculatedTaxSummaryForService (
  numberOfLines: Option[Integer],  // Count of lines
subtotal: Option[Double],  // sum of all line tax attribute
totalTax: Option[Double],  // sum of all line lineAmount attribute
grandTotal: Option[Double],  // sum of all line lineAmount attribute - sum of all line tax attribute - sum of all line lineTaxedDiscount attribute
taxByType: Option[SalesCalculatedTaxSummaryForService_taxByType]
)

File: javascript-closure-angular-client/API/Client/SalesCalculatedTaxSummaryForService.js
goog.provide('API.Client.SalesCalculatedTaxSummaryForService');

/**
 * @record
 */
API.Client.SalesCalculatedTaxSummaryForService = function() {}

/**
 * Count of lines
 * @type {!number}
 * @export
 */
API.Client.SalesCalculatedTaxSummaryForService.prototype.numberOfLines;

/**
 * sum of all line tax attribute
 * @type {!number}
 * @export
 */
API.Client.SalesCalculatedTaxSummaryForService.prototype.subtotal;

/**
 * sum of all line lineAmount attribute
 * @type {!number}
 * @export
 */
API.Client.SalesCalculatedTaxSummaryForService.prototype.totalTax;

/**
 * sum of all line lineAmount attribute - sum of all line tax attribute - sum of all line lineTaxedDiscount attribute
 * @type {!number}
 * @export
 */
API.Client.SalesCalculatedTaxSummaryForService.prototype.grandTotal;

/**
 * @type {!API.Client.SalesCalculatedTaxSummaryForService_taxByType}
 * @export
 */
API.Client.SalesCalculatedTaxSummaryForService.prototype.taxByType;

